src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginCache.cs

```diff
@@ -46,6 +47,8 @@ public class TestPluginCache
         /// </summary>
         private AssemblyResolver assemblyResolver;
 
+        private IAssemblyResolver platformAssemblyResolver;
```

Comment:
Create assemblyResolver objects in internal .ctor, to enable mocking in unit tests.